packages/auth/amplify_auth_cognito_dart/lib/src/auth_plugin_impl.dart

```diff
@@ -445,17 +447,34 @@ class AmplifyAuthCognitoDart extends AuthPluginInterface
           continue;
         case SignInStateType.challenge:
           state as SignInChallenge;
-          return SignInResult(
-            isSignedIn: false,
-            nextStep: AuthNextSignInStep(
-              signInStep: state.challengeName.signInStep.value,
-              codeDeliveryDetails: _getChallengeDeliveryDetails(
-                state.challengeParameters,
-              ),
-              challengeParameters: state.challengeParameters,
-              missingAttributes: state.requiredAttributes,
-            ),
-          );
+
+          switch (state.challengeName) {
```

Comment:
We should just get rid of `challengeParameters`. I added it because I didn't realize we had `additionalInfo`

Reply:
Done

Reply:
Let's actually remove it from the result object I mean, since I added them only recently